src/main/java/de/medizininformatik_initiative/process/report/spring/config/FhirClientConfig.java

```diff
@@ -152,6 +152,11 @@ public class FhirClientConfig
 	@Value("${de.medizininformatik.initiative.report.dic.fhir.dataLoggingEnabled:false}")
 	private boolean fhirDataLoggingEnabled;
 
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "Initial Result polling interval in milliseconds for asynchronous request pattern when executing search bundle requests, the interval will be increased tenfold after every check if a result is not ready")
```

Comment:
Is "tenfold" correct?

https://github.com/medizininformatik-initiative/mii-processes-common/blob/0ad23ab4cd2a6b09af359bec32f2cb20bc579c4f/src/main/java/de/medizininformatik_initiative/processes/common/fhir/client/AsyncFhirClientImpl.java#L56

```suggestion
			"medizininformatik-initiativede_reportSend" }, description = "Initial result polling interval in milliseconds for asynchronous request pattern when executing search bundle requests, the interval will double after every check if a result is not ready")
```

Reply:
Implemented in 2c7ca66